package.json

```diff
@@ -102,16 +102,20 @@
     "victory": "^32.2.3",
     "webpack": "^4.5.0",
     "webpack-cli": "^2.0.14",
-    "webpack-log": "^2.0.0"
+    "webpack-log": "^2.0.0",
+    "xstate": "^4.7.7"
   },
   "devDependencies": {
     "@babel/plugin-syntax-decorators": "^7.2.0",
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
     "@babel/plugin-syntax-jsx": "^7.2.0",
     "@babel/plugin-syntax-typescript": "^7.2.0",
+    "@testing-library/jest-dom": "^5.0.2",
+    "@testing-library/react": "^9.4.0",
     "@types/enzyme": "^3.1.15",
     "@types/enzyme-adapter-react-16": "^1.0.3",
     "@types/jest": "^23.1.1",
+    "@xstate/test": "^0.3.0",
```

Comment:
We will want to start locking down the dependency versions. Instead of `"@xstate/test": "^0.3.0"`, simply use "0.3.0"